api/envoy/config/metrics/v3/stats_circonus.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```

Comment:
I think just naming this `circonus.proto` is fine.

Reply:
That said, it seems kind of strange we are putting custom extensions in this tree vs `envoy/extensions`. I think this was grandfathered in due to the `stats.proto` monolith. Would we better off just keeping it in `stats.proto` or splitting out into a new extension tree? @envoyproxy/api-shepherds

Reply:
+1 to new extensions tree. Let's try to do the right thing for new stuff?

Reply:
Can you guys please tell me the directory you want me to use?  Thanks

Reply:
I think the intention is that this proto should go in `api/envoy/extensions/stats_sinks/circonus`.

Reply:
Ah ok, thanks

Reply:
@zuercher @mattklein123 

They're two protobuf files, one for v3 and another for v4 alpha.  You mentioned you wanted it in the circonus directory directly but what are you trying to establish as the standard for versions when it's shipped directly with the code?